common/src/main/java/com/wynntils/core/consumers/features/FeatureManager.java

```diff
@@ -452,6 +454,12 @@ public void disableFeature(Feature feature) {
         Managers.Overlay.disableOverlays(feature);
 
         Managers.KeyBind.disableFeatureKeyBinds(feature);
+
+        if (McUtils.mc().screen instanceof WynntilsBookSettingsScreen settingsScreen) {
+            ConfigTile configTile = settingsScreen.getConfigurableStateButton(feature);
+            if (configTile == null) return;
+            configTile.getConfigOptionElement().onClick(0, 0);
```

Comment:
Also, this looks fragile, assuming that you will always toggle, and faking a click. It is better to propagate the request to make sure the tile refreshes its state.

Reply:
In fact; this fix will probably overshadow the discord one. So it's probably better that you separate this into two PRs.